test/backend/listingAction.js

```diff
@@ -9,10 +9,54 @@ describe('listingAction', () => {
         })
     })
 
+    it('id should be a number', () => {
+        return listingAction.promiseMeTheListing()
+        .then(result => {
+          for (var i = 0; i < result.length; i++) {
+            (isNaN(result[0].id)).should.be.equal(false)
```

Comment:
w razie faila dostaniesz `expected true to be false`
```
result[0].should.have.property("id")
result[0]id.should.be.type("number")
```
a to Ci powie dokładnie ci się stało